summary/table.go

```diff
@@ -41,7 +42,13 @@ func DisplaySummaryOutput(resp *opb.Response, revDNSLookup bool, requestType str
 				procStrSlice = append(procStrSlice, procData.ProcName)
 				procStrSlice = append(procStrSlice, procData.Count)
 				procStrSlice = append(procStrSlice, procData.UpdatedTime)
-				procStrSlice = append(procStrSlice, procData.Status)
+				agc := ansi.ColorFunc("green")
```

Comment:
@yasin-cs-ko-ak Please refrain from calling in multiple places. Call in one place and use it here.

Reply:
@seswarrajan Thank you for reviewing. I updated the changes.